lib/src/node/legacy.dart

```diff
@@ -210,7 +210,7 @@ List<AsyncCallable> _parseFunctions(RenderOptions options, DateTime start,
 
     var fiber = options.fiber;
     if (fiber != null) {
-      result.add(Callable.fromSignature(signature, (arguments) {
+      result.add(Callable.fromSignature(signature.trimLeft(), (arguments) {
```

Comment:
There are two more places below in legacy API, do we need to match all three calls? Not sure what is the scope of node-sass bug we’re trying to match.